Problems with pip installation of the new napari version

Trying to pip install napari in anaconda 3 on a Windows produces an error.


## To Reproduce

Steps to reproduce the behavior:

1. From terminal, `pip install napari` gives me:

```
ERROR: Exception:
Traceback (most recent call last):
  File "C:\Users\lpetrucco\anaconda3\lib\site-packages\pip\_internal\cli\base_command.py", line 188, in main
    status = self.run(options, args)
  File "C:\Users\lpetrucco\anaconda3\lib\site-packages\pip\_internal\commands\install.py", line 407, in run
    use_user_site=options.use_user_site,
  File "C:\Users\lpetrucco\anaconda3\lib\site-packages\pip\_internal\req\__init__.py", line 58, in install_given_reqs
    **kwargs
  File "C:\Users\lpetrucco\anaconda3\lib\site-packages\pip\_internal\req\req_install.py", line 928, in install
    use_user_site=use_user_site, pycompile=pycompile,
  File "C:\Users\lpetrucco\anaconda3\lib\site-packages\pip\_internal\req\req_install.py", line 461, in move_wheel_files
    warn_script_location=warn_script_location,
  File "C:\Users\lpetrucco\anaconda3\lib\site-packages\pip\_internal\wheel.py", line 432, in move_wheel_files
    clobber(source, lib_dir, True)
  File "C:\Users\lpetrucco\anaconda3\lib\site-packages\pip\_internal\wheel.py", line 381, in clobber
    ', '.join(info_dir))
AssertionError: Multiple .dist-info directories: C:\Users\lpetrucco\anaconda3\Lib\site-packages\napari\plugins\_tests\fixtures\napari_working_plugin-0.1.0.dist-info, C:\Users\lpetrucco\anaconda3\Lib\site-packages\napari-0.2.12.dist-info
```



I have tried also in a completely new conda env and the same happens. I was able to install using conda-forge.

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,6 +3,7 @@ language: python
 python:
   - "3.7.5"
 install:
+  - pip install --upgrade pip
   - pip install -r REQUIREMENTS.txt
   - pip install -r REQUIREMENTS-DEV.txt
   - pip install -e .

